ha_reolink_rich_notifications.yaml

```diff
@@ -36,13 +36,31 @@ blueprint:
       selector:
         duration:
 
+    notification_title:
+      name: Notification Heading
+      description: Title of the notification
```

Comment:
Could you add another sentence here, that if left blank, will default to the name of the camera?